addons/website/tests/test_performance.py

```diff
@@ -34,7 +34,7 @@ def test_10_perf_sql_img_controller(self):
         url = '/web/image/res.country/4/image'
         self.assertEqual(self._get_url_hot_query(url), 4)
         url = '/web/image/res.users/2/image_256'
-        self.assertEqual(self._get_url_hot_query(url), 7)
+        self.assertEqual(self._get_url_hot_query(url), 8)
```

Comment:
Hello

Did you check this new query ? 
Because for a strange reason, I cannot see this extra query in local...

All what I can see is 

that 

```sql
SELECT "res_users"."id" as "id","res_users"."partner_id" as "partner_id","res_users"."login" as "login","res_users"."signature" as "signature","res_users"."active" as "active","res_users"."action_id" as "action_id","res_users"."share" as "share","res_users"."company_id" as "company_id","res_users"."notification_type" as "notification_type","res_users"."website_id" as "website_id","res_users"."create_uid" as "create_uid","res_users"."create_date" as "create_date","res_users"."write_uid" as "write_uid","res_users"."write_date" as "write_date" FROM "res_users","res_partner" as "res_users__partner_id" WHERE "res_users".id IN (2) AND (("res_users"."share" IS NULL or "res_users"."share" = false ) OR ("res_users"."id" in (SELECT "user_id" FROM "res_company_users_rel" WHERE "cid" IN (1)))) AND ("res_users"."partner_id" = "res_users__partner_id"."id") AND (((("res_users__partner_id"."partner_share" IS NULL or "res_users__partner_id"."partner_share" = false ) OR ("res_users__partner_id"."company_id" in (1))) OR "res_users__partner_id"."company_id" IS NULL ) AND ((("res_users__partner_id"."type" != 'private') OR "res_users__partner_id"."type" IS NULL) OR "res_users__partner_id"."type" IS NULL ))
```

became

```sql
SELECT "res_users"."id" as "id","res_users"."partner_id" as "partner_id","res_users"."login" as "login","res_users"."signature" as "signature","res_users"."active" as "active","res_users"."action_id" as "action_id","res_users"."share" as "share","res_users"."company_id" as "company_id","res_users"."notification_type" as "notification_type","res_users"."website_id" as "website_id","res_users"."create_uid" as "create_uid","res_users"."create_date" as "create_date","res_users"."write_uid" as "write_uid","res_users"."write_date" as "write_date" FROM "res_users" WHERE "res_users".id IN (2)
```